### PR TITLE
ci: Create Windows Build script and Move poro.ico 

### PR DESCRIPTION
**Pull request by alepouna**
* Creates a Windows Build Script to allow building the exe in Windows
* Move the poro.ico in src/

### CODE REVIEW

**Comment by auroraisluna** [comment]
Seems like GitHub added all my previous commits as well in the history.. interesting. Refer to 'file changes' for changes.

File: setup/windows.bat [hunk 1]
```diff
@@ -0,0 +1,146 @@
+@echo off
+@REM [BETA] v
```
Comment (hypixus) [inline setup/windows.bat:2]: I believe comments within .BAT files are made by simply using `REM <comment>`.

File: setup/windows.bat [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+echo.
+echo CapsuleFarmerEvolved Windows Build Script
+echo.
+echo ========================================
+echo.
+echo Building application...
+echo. 
+
+@REM Now we need to build the application with pipenv 
+python -m pipenv run pyinstaller -F --icon=src/poro.ico src/main.py --collect-all charset_normalizer -n CapsuleFarmerEvolved
```
Comment (hypixus) [inline setup/windows.bat:44]: This assumes default version on local machine is 3, which is not always true. Please use `python3`.

File: setup/windows.bat [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+echo CapsuleFarmerEvolved Windows Build Script
+echo.
+echo ========================================
+echo.
+echo Would you like to create a zip file of the application? (Requires 7zip on PATH) [Y/N]
+echo.
+
+set /p createZipFile=
+
+set zipFileAnswer=no
```
Comment (hypixus) [inline setup/windows.bat:71]: How about using `true`/`false`? Batch does not understand boolean values, but that keeps the clarity in the longer run.

File: setup/windows.bat [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+)
+
+cls
+echo ========================================
+echo.
+echo CapsuleFarmerEvolved Windows Build Script
+echo.
+echo ========================================
+echo.
+echo [OK] Application build complete. You can find the application in the build directory.
```
Comment (hypixus) [inline setup/windows.bat:140]: The section with resetting the console window above is repeated a lot. How about throwing it into [a proper batch function?](http://www.trytoprogram.com/batch-file-functions/)

**Review by hypixus — CHANGES_REQUESTED** [review]

File: setup/windows.bat (hunk 2 above)
Comment (auroraisluna) [reply setup/windows.bat:44]: There are Prerequisites to build the app (listed in the wiki), it is assumed these are followed. Advanced users modify the script as required.

File: setup/windows.bat (hunk 1 above)
Comment (auroraisluna) [reply setup/windows.bat:2]: @REM and REM are the same commands, the @REM ensures the comment isn't echoed, regardless of the echo on/off state. As intended.

File: setup/windows.bat (hunk 3 above)
Comment (auroraisluna) [reply setup/windows.bat:71]: Didn't think of using true/false at the time as like you mentioned batch does not read booleans, only 32bit ints or strings. Using `on/off` have the same meaning both linguistic and in variable type, I don't see how it adds more clarity.

**Comment by LeagueOfPoro** [comment]
Let's move the `poro.ico` to `src/assets/poro.ico` to match #80

**Review by hypixus — APPROVED** [review]

**Comment by DjordyKoert** [comment]
> Let's move the `poro.ico` to `src/assets/poro.ico` to match #80

I think `assets/poro.ico` is a better spot for these kind of  files. 
The banner image can also be placed here most likely.

File: setup/windows.bat (hunk 2 above)
Comment (auroraisluna) [reply setup/windows.bat]: I will repeat my self more clearly this time:
The prerequisites demand for Python 3.10, if your system has different versions installed, you are not following the prerequisites, and therefore you will need to edit the script on your own to match them.

This is an advanced script aimed at advanced users. 
If said advanced user doesn't know the different Python commands in their system, that's on them. 

And by the way, Python3 isn't a default either. Assuming the mentioned points didn't matter, it would still not work for a lot of advanced users. 

TLDR; Useless back-and-forth for a fix each advanced user can apply as required up to their current installation.

**Comment by auroraisluna** [comment]
Will not be moving the banner. It is not an asset of the app.
